.github/workflows/ci.yml

```diff
@@ -7,10 +7,11 @@ on:
     branches: [main]
 
 jobs:
-  build:
+  main:
```

Comment:
Do you mind leaving this job name as `build`?

Reply:
Yes sure